starter-core/src/test/groovy/io/micronaut/starter/feature/messaging/kafka/KafkaSpec.groovy

```diff
@@ -62,7 +62,7 @@ class KafkaSpec extends ApplicationContextSpec implements CommandOutputFixture {
         template.contains('''<artifactId>micronaut-maven-plugin</artifactId>
           <configuration>
             <shared>true</shared>
-          </configuration>''')
```

Comment:
I had to remove this because config with AOT enabled looks like: 

```xml
    <artifactId>micronaut-maven-plugin</artifactId>
          <configuration>
            <shared>true</shared>
            <configFile>aot-${packaging}.properties</configFile>
          </configuration>
```